samples/msal-angular-v2-samples/angular10-sample-app/src/app/app.module.ts

```diff
@@ -25,10 +25,8 @@ export function loggerCallback(logLevel: LogLevel, message: string) {
 export function MSALInstanceFactory(): IPublicClientApplication {
   return new PublicClientApplication({
     auth: {
-      // clientId: '6226576d-37e9-49eb-b201-ec1eeb0029b6', // Prod enviroment. Uncomment to use. 
-      clientId: '3fba556e-5d4a-48e3-8e1a-fd57c12cb82e', // PPE testing environment
-      // authority: 'https://login.microsoftonline.com/common', // Prod environment. Uncomment to use.
-      authority: 'https://login.windows-ppe.net/common', // PPE testing environment.
+      clientId: 'ENTER_CLIENT_ID_HERE',
+      authority: 'https://login.microsoftonline.com/common',
```

Comment:
We may want to add a comment here pointing to app registration.  Not sure that common will work for apps that are registered as single tenant.  I don't recall though.

Reply:
fair point -just added